components/announcement_bar/announcement_bar.jsx

```diff
@@ -234,15 +234,15 @@ export default class AnnouncementBar extends React.PureComponent {
         AnalyticsStore.removeChangeListener(this.onAnalyticsChange);
     }
 
-    componentDidUpdate(prevProps, prevState) {
+    componentDidUpdate() {
```

Comment:
Resisting the urge to do much refactoring here: just trying to centralize the logic that toggles this class.